src/summary/documentation/dimensions-card-level-two.component.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import styles from "./dimensions-card-level-two.css";
+import SummaryCard from "../cards/summary-card.component";
+import { getDimensions } from "./dimensions-card.resource";
+import { match } from "react-router";
+
+function DimensionsCardLevelTwo(props: DimensionsCardLevelTwoProps) {
+  const [dimensions, setDimensions] = React.useState([]);
+
+  React.useEffect(() => {
+    const sub = getDimensions(props.currentPatient.id).subscribe(dimensions =>
+      setDimensions(dimensions)
+    );
+    return () => sub.unsubscribe();
+  }, []);
+
+  return (
+    <div style={{ display: "flex" }}>
+      <SummaryCard
+        name="Height & Weight"
+        match={props.match}
+        styles={{ flex: 1, margin: ".5rem" }}
+      >
+        <table className={styles.table}>
+          <thead>
+            <tr className={styles.tableRow}>
+              <th
+                className={`${styles.tableHeader} ${styles.tableDates}`}
+                style={{ textAlign: "start" }}
```

Comment:
Couldn't this just be added to the already existing `tableDates` class?

Reply:
done